File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/StealIdResponseContent.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

public class StealIdResponseContent extends BaseResponseContent {

    @SerializedName("comm_key")
    public String commKey;

    @SerializedName("device_name")
    public String deviceName;

    @SerializedName("device_suid")
    public String deviceSuid;
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/StealIdResponseContent.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

class StealIdResponseContent : BaseResponseContent() {

    @SerializedName("comm_key")
    var commKey: String? = null

    //device_name=FM-Mobile-Entry 800
    @SerializedName("device_name")
    var deviceName: String? = null


    //device_suid=dim.d.800
    @SerializedName("device_suid")
    var deviceSuid: String? = null
}